ESCPOS_NET/Printers/SerialPrinter.cs

```diff
@@ -1,24 +1,19 @@
-﻿using System;
 using System.IO;
 using System.IO.Ports;
 using System.Threading.Tasks;
 
 namespace ESCPOS_NET
 {
-    public class SerialPrinter : BasePrinter,  IDisposable
```

Comment:
Can we not implement idisposable anymore or is there some reason not to? this is something I'm not super familiar with

Reply:
This is done by the base so no need to have it in the child classes